src/graphql/util.js

```diff
@@ -34,7 +34,7 @@ export function getOperatorAndOperand(expression) {
   } else if (typeof expression.GT !== 'undefined') {
     return { operator: '>', operand: expression.GT };
   }
-  return {};
+  throw new Error('Invalid Expression!');
```

Comment:
I think we should throw an exception here. If `{}` is returned, it still causes an error when conducting queries.